fennel/lib/to_proto/to_proto.py

```diff
@@ -1173,25 +1173,21 @@ def _kinesis_to_ext_db_proto(
 def _kinesis_to_ext_table_proto(
     db: connector_proto.ExtDatabase,
     stream_arn: str,
-    init_position: sources.InitPosition,
-    init_timestamp: Optional[datetime] = None,
+    init_position: str | kinesis.at_timestamp,
     format: str = "json",
```

Comment:
should we remove default value from here too and add a validation that format must be json?

Reply:
We only need to make the argument on the public API optional and have a default value. As the value propagates down the code to protos, we shouldn't have default values? But rather add validation that it is not json?

Reply:
the validation is already present and tested but good catch, will remove this here

Reply:
I don't think there is a validation added here in this method (to_proto.py file for this). Please add one and add an unit test for it in test_to_proto.py (this future proofs that if there is a different caller for this, which there could easily be, fails if they pass an invalid format).